misc/PFS_Abbreviations.tsv

```diff
@@ -38,86 +39,91 @@ DRP	Software	General	Data Reduction Pipeline	Software for data processing. There
 FFF	PFI	Subcomponent	Fixed Fiducial Fibers	Fibers fixed on the Cobra bench (COB) used as a reference of fiber position. There are 96 FFF in total, 84 for science fibers, and 12 for AG cameras.
+FFI	PFI	Subcomponent	Fiducial Fiber Illuminator	PFI component to back-illuminate fixed fiducial fibers and take image of the light coming through the fiducial fibers.
 FIA	SpS	Subcomponent	Fiber Input sub-Assembly	FOCCoS assembly as a part of SpS to connect Cable A with Cable B. LNA has a responsibility for its development.
 FIS	PFI	Subcomponent	Fiducial Fiber Illumination Source	The system to back-illuminate fixed fiducial fibers.
 FIT	SpS	Subcomponent	Focal Plane Assembly integration tool	AIT tool of SpS Camera Assembly (CAA). FIT is used for integrating Focal Plane assembly into camera optics.
 FMCS	SpS	Subcomponent	Focus Mechanism Control System	TBW.
 fms	ICS	Actor	Fiber connection Monitoring System	FOCCoS software to illuminate monitoring fibers, to take their image, and judge the connection among Cable A, B and C.
 FOCCoS	General	PFS Subsystem	Fiber Optical Cable and Connector System	PFS instrument subsystem which delivers light from PFI at the prime focus to SpS  on the IR4 floor. LNA has a responsibility for FOCCoS development.
 FPA	SpS	Subcomponent	Focal Plane Assembly	SpS assembly in camera unit. FPA consists of detectors and focusing mechanism.
-FPRD	General	Document	Functional Performance Requirements Document	One of the documents at Preliminary Design Review. FPRD describes all requirements for PFS instrument to sartisfy scientific requirements. For system-level requirements, the latest version is available on PBworks.
-FPS	ICS	Actor	Fiber Positioning Sequencer	PFI software to operate fiber configuration. FPS controls MCS exposure, derives fibers positions from measured centroids, and sends them with command to MPS to move fibers. ASIAA has a responsibility for its development.
+FPRD	General	Document	Functional Performance Requirements Document	One of the documents at Preliminary Design Review. FPRD describes all requirements for PFS instrument to satisfy scientific requirements. For system-level requirements, the latest version is available on PBworks.
+fps	ICS	Actor	Fiber Positioning Sequencer	PFI software to operate fiber configuration. FPS controls MCS exposure, derives fibers positions from measured centroids, and sends them with command to MPS to move fibers. ASIAA has a responsibility for its development.
 FPSA	SpS	Subcomponent	Fiber Slit Positioning Sub-Assembly	SpS assembly to position fiber slit. FPSA consists of Hexapod to tilt and shift the Fiber Slit sub-Assembly (FSA).
-FSA	SpS/FOCCoS	Subcomponent	Fiber Slit Assembly	FOCCos assembly as a part of SpS. The slit accommodates about 600 fibers in a row.
+FSA	SpS/FOCCoS	Subcomponent	Fiber Slit Assembly	Focus assembly as a part of SpS. The slit accommodates about 600 fibers in a row.
```

Comment:
focus or fiber slit? focus assembly seems to include hex-stage..

Reply:
Ah, thank you for pointing.
Recently SpS team discussed and FSA is now used for "Fiber Slit Sub-Assembly"
Hex staged one is called FPSA or FPA (Fiber Slit Positioning Sub-Assembly)
I should have modified taking these updates into account.